styleguide/components/Scaffold/Scaffold.css

```diff
@@ -66,6 +66,28 @@
   flex-basis: 100%;
 }
 
+.blockquote {
+  margin: 0;
+  padding: var(--size-regular);
+  border-left: 0.25rem solid var(--color-greyDark);
+}
+
+.blockquote .showQuotationMarks:before {
+  position: relative;
+}
+
+.blockquote .showQuotationMarks:before {
+  top: 0;
+  left: 0;
+  content: "\201C";
```

Comment:
I don't think we need to have the quotation marks any case. The styling should be enough